dataservice/api/participant/resources.py

```diff
@@ -16,8 +16,15 @@ def get(self, kf_id):
         Gets a participant given a Kids First id
         """
         if kf_id is None:
+            def_size = current_app.config['DEFAULT_PAGE_SIZE']
+            max_size = current_app.config['MAX_PAGE_SIZE']
+            size = min(request.args.get('size', def_size, type=int), max_size)
+            page = request.args.get('page', 1, type=int)
+
+            q = Participant.query.order_by(Participant.created_at.desc())
```

Comment:
Uses `limit() offset()`, so not ideal. Will have to implement our own pagination later using `limit() filter()` on the created_at field.